enterprise/README.md

```diff
@@ -1,33 +1,18 @@
-# Official Flyway Teams and Enterprise Docker images
+# Official Flyway Community, Teams and Enterprise Docker images
 
-[![Docker Auto Build](https://img.shields.io/docker/cloud/automated/redgate/flyway)][docker]
+This is the official repository for [Flyway Command-line](https://documentation.red-gate.com/fd/welcome-to-flyway-184127914.html) images.
 
-[docker]: https://hub.docker.com/r/redgate/flyway/
 
-This is the official repository for [Flyway Command-line](https://flywaydb.org/documentation/usage/commandline/) images.
+## Suggested Project Structure
 
-The Flyway Teams and Enterprise images are available in [redgate/flyway](https://hub.docker.com/r/redgate/flyway/) on Dockerhub.
+To make it easy to run Flyway the way you want to, you can use the following folders in your Flyway project
 
-## Supported Tags
-
-The following tags are officially supported:
-
-- [`9.2.1`, `9.2`, `9`, `latest` (*Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/Dockerfile)
-- [`9.2.1-alpine`, `9.2-alpine`, `9-alpine`, `latest-alpine` (*alpine/Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/alpine/Dockerfile)
-- [`9.2.1-azure`, `9.2-azure`, `9-azure`, `latest-azure` (*azure/Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/azure/Dockerfile)
-
-The **redgate/flyway:\*-azure** images *only* support alpine versions.
-
-## Supported Volumes
-
-To make it easy to run Flyway the way you want to, the following volumes are supported:
-
-Volume            | Usage
+Folder            | Usage
 ------------------|------
-`/flyway/conf`    | Directory containing a `flyway.conf` [configuration file](https://flywaydb.org/documentation/usage/commandline/#configuration)
-`/flyway/drivers` | Directory containing the [JDBC driver for your database](https://flywaydb.org/documentation/usage/commandline/#jdbc-drivers)
-`/flyway/sql`     | The SQL files that you want Flyway to use (for [SQL-based migrations](https://flywaydb.org/documentation/concepts/migrations#sql-based-migrations))
-`/flyway/jars`    | The jars files that you want Flyway to use (for [Java-based migrations](https://flywaydb.org/documentation/concepts/migrations#java-based-migrations))
+`/conf`    | Directory containing a `flyway.conf/toml` [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
```

Comment:
```suggestion
`/flyway/conf`    | Directory containing a `flyway.conf/toml` [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
```

Should these remain `/flyway/x` like the OSS image?

Reply:
It was an oversight on the flyway/flyway page - originally these were individual volume mounts but it makes it ungainly to use docker & flyway that way. The recent workingdirectory param updates should mean you just mount the project folder and go from there which is more in line with Desktop. I'm wondering if this ought to be pulled out of here and put into the docs pages

Reply:
Decided to put it back the way it was and just make the changes Adam asked for originally. We can make further changes later !